packages/clay-button/src/ClayButton.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+
+export const DISPLAY_TYPES = ['primary', 'secondary', 'link', 'unstyled'];
+
+interface Props extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+	block?: boolean;
+	displayType?: typeof DISPLAY_TYPES[number];
```

Comment:
The main reason I use a constant is so that users can import the types and see what is available. If the user is using TS, they wont need it because they an infer what is available, but someone is using regular js, they can import to see what types are available. This may be overkill though...

Reply:
I understand your concern about this, but we do not have to worry when people are not using TS if we can not fill our components with utilities... we can delegate that responsibility to documentation.